scripts/package.sh

```diff
@@ -27,22 +27,50 @@ if [ ! -f "${tmp_license_file}" ]; then
   echo "Cannot find a LICENSE file called ${filename} in the exporter repo";
   exit 1;
 fi
-cp "${tmp_license_file}" "${exporters_doc_dir}/${integration}-LICENSE"
+cp "${tmp_license_file}" "${integration_target}/${integration}-LICENSE"
 rm -rf "${tmp_dir}"
 
 echo "Packaging"
+mkdir -pv "${integration_target}/packages"
 if [ "$goos" == "windows" ]; then
-  powershell.exe -file "${root_dir}/scripts/win_msi_build.ps1" -arch amd64 -exporterName ${NAME} -version ${VERSION} -exporterGUID ${EXPORTER_GUID} -upgradeGUID ${UPGRADE_GUID} -licenseGUID ${LICENSE_GUID}
+    powershell.exe -file "${root_dir}/scripts/win_msi_build.ps1" -arch amd64 -exporterName ${NAME} -version ${VERSION} -exporterGUID ${EXPORTER_GUID} -upgradeGUID ${UPGRADE_GUID} -licenseGUID ${LICENSE_GUID}
+
+    mkdir -p "${root_dir}/dist/New Relic/newrelic-infra/integrations.d"
+    cp "${integration_dir}/${NAME}-config.yml.sample" "${root_dir}/dist/New Relic/newrelic-infra/integrations.d/"
+
+    mkdir -p "${root_dir}/dist/New Relic/newrelic-infra/newrelic-integrations/bin"
+    cp "${integration_target}/bin/nri-${NAME}.exe" "${root_dir}/dist/New Relic/newrelic-infra/newrelic-integrations/bin"
+    cp "${integration_dir}/${NAME}-win-definition.yml" "${root_dir}/dist/New Relic/newrelic-infra/newrelic-integrations/" || true
+
+    mkdir -p "${root_dir}/dist/New Relic/newrelic-infra/logging.d"
+    cp "${integration_dir}/${NAME}-log-win.yml" "${root_dir}/dist/New Relic/newrelic-infra/logging.d/" || true
+
+    (
+      # Inside a subshell so we do not change $PWD to the rest of the script
+      cd "${root_dir}/dist"
+      7z a -tzip "${integration_dir}/target/packages/nri-${NAME}-amd64.${VERSION}.zip" "New Relic"
+    )
 else
-   cp "${goreleaser_file_template}" "${goreleaser_file}"
-   IFS=',' read -r -a goarchs <<< "$PACKAGE_LINUX_GOARCHS"
-   for goarch in "${goarchs[@]}"
-   do
-     echo  "Adding ${goarch} to goreleaser"
-     yq e -i ".builds[0].goarch += [ \"${goarch}\" ]" "${goreleaser_file}"
-   done
-
-   GORELEASER_CURRENT_TAG=${VERSION} ${goreleaser_bin} release --config "${goreleaser_file}" --rm-dist --snapshot
-   echo "Signing the packages"
-   bash ${root_dir}/scripts/sign.sh "${root_dir}" "${integration}"
+    cp "${goreleaser_file_template}" "${goreleaser_file}"
+    yq e -i ".nfpms[0].package_name = \"nri-${NAME}\"" "${goreleaser_file}"
+
+    IFS=',' read -r -a goarchs <<< "$PACKAGE_LINUX_GOARCHS"
+    for goarch in "${goarchs[@]}"
+    do
+      echo  "Adding ${goarch} to goreleaser"
+      yq e -i ".builds[0].goarch += [ \"${goarch}\" ]" "${goreleaser_file}"
+    done
+
+    if [ "x${CI}" = "xtrue" ]; then
+      git tag ${VERSION}  # Tag the repository in case we are in a CI.
+      GORELEASER_CURRENT_TAG=${VERSION} ${goreleaser_bin} release --config "${goreleaser_file}" --rm-dist
```

Comment:
I might leave a comment here too...

This is because if we package a `deb` or a `rpm` with `--snapshot` the version of the package is something like `${GORELEASER_CURRENT_TAG}-SNAPSHOT-${SHORT_SHA1}`.

goreleaser does not allow overriding it in any way (that I could find) so I tagged the repo (locally) to be able to use that tag.

It is made this way only in a CI because locally you cannot test properly as the repository usually is dirty while you are developing.

Reply:
I included a comment to clarify why we only use snapshot running locally. However, now I wonder if the `git tag ${VERSION}` is needed. I think we are already using [the way to override the tag](https://goreleaser.com/cookbooks/set-a-custom-git-tag/) using `GORELEASER_CURRENT_TAG`.

Did you find any problems before adding it?

Reply:
Thanks for sharing all the context offline. I included additional comments here: [82fb659](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/147/commits/82fb659ad21cd49e6e96d319ad902d10fac52b99)